batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/constants/Url.java

```diff
@@ -0,0 +1,15 @@
+package com.lftechnology.batch7crud.constants;
+
+/**
+ * Created by sanjay on 1/21/16.
+ */
+public class Url {
+  public static final String CREATE = "create";
+  public static final String VIEW = "view";
+  public static final String EDIT = "edit";
+  public static final String DELETE = "delete";
+  public static final String STUDENTS = "students";
+
+  private Url(){}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)